frontend/src/renderer/components/Process/Dropbox.svelte

```diff
@@ -16,20 +16,20 @@
     }
 
 </script>
-<main>
-    <div class="sectionStyle {showError && singleItem && isEmpty(items) ? 'errorStyle' : ''}">
-        <section use:dndzone={{items, flipDurationMs, type, dropFromOthersDisabled: singleItem && items.length > 0}}
-            on:consider={handleSort}
-            on:finalize={handleSort}
-        >
-            {#each items as item (item?.id)}
-                <card animate:flip={{ duration: flipDurationMs }} class="{singleItem ? 'verified' : ''}">
-                    <ImageBubble filename={item.name} minimal />
-                </card>
-            {/each}
-        </section>
-    </div>
-</main>
+
+<div class="sectionStyle {showError && singleItem && isEmpty(items) ? 'errorStyle' : ''}">
```

Comment:
is `showError` unused?

Reply:
Actually it looks like this is used, but for some reason the 'errorStyle' doesn't get applied
- [ ] Either apply errorStyle or remove the code that modifies it